internal/guest/runtime/hcsv2/container.go

```diff
@@ -27,6 +27,13 @@ import (
 	"github.com/Microsoft/hcsshim/internal/protocol/guestresource"
 )
 
+type containerStatus string
+
+const (
+	containerCreating containerStatus = "Creating"
+	containerRunning  containerStatus = "Running"
+)
+
```

Comment:
If we don't need to print these as strings and they're just used for state checks, probably better to have these just be uint8's/a numeric type